### ISSUE TITLE
where is the gradient

### ISSUE THREAD

**Issue by panjianfei** [microsoft/DeepSpeed#4682]
**Describe the bug**
parameter gradient is None after deepspeed.initialize()

```
p model_engine.lm_head.weight
tensor([], device='cuda:0', dtype=torch.bfloat16, requires_grad=True)
```

**Comment by zouyingcao** [comment]
Sad... I have the same issue. Did you solve it?
